github/ci/prow/templates/mirror-proxy-service.yaml

```diff
@@ -0,0 +1,9 @@
+apiVersion: v1
+kind: Service
+metadata:
+  name: docker-mirror-proxy
+spec:
+  selector:
+    app: docker-mirror-proxy
+  ports:
+  - port: 3128
```

Comment:
The proxy uses a single port for both http and https, from the PoV of the daemons this will be transparent, they will still use the designated ports for connecting with the target registries, sorry but I don't see clearly what is the benefit of declaring these different ports.

Reply:
Makes sense to me now.